docs/openapi/api-internal-pn-bff-pg-apikeys.yaml

```diff
@@ -0,0 +1,495 @@
+        - $ref: './common-refs.yaml#/components/parameters/cxRoleAuthFleet'
+        - $ref: '#/components/parameters/pathVirtualKeyId'
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Bad request
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+        '409':
+          description: Wrong state transition
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+        '404':
+          description: Not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+        '500':
+          description: Internal error
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+
+  "/virtual-keys/{id}/status":
+    put:
+      summary: Cambia lo stato della virtualKey
+      description: >-
+        servizio di cambio stato della virtualKey
+      tags:
+        - VirtualKeys
+      operationId: changeStatusVirtualKeysV1
+      parameters:
+        - $ref: './common-refs.yaml#/components/parameters/uidAuthFleet'
+        - $ref: './common-refs.yaml#/components/parameters/cxTypeAuthFleet'
+        - $ref: './common-refs.yaml#/components/parameters/cxIdAuthFleet'
+        - $ref: './common-refs.yaml#/components/parameters/cxGroupsAuthFleet'
+        - $ref: './common-refs.yaml#/components/parameters/cxRoleAuthFleet'
+        - $ref: '#/components/parameters/pathVirtualKeyId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "./virtualKeys-schemas/virtual-keys.yaml#/components/schemas/BffVirtualKeyStatusRequest"
+        required: true
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Bad request
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+        '409':
+          description: Wrong state transition (i.e. enable an enabled key)
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+        '404':
+          description: Not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+        '500':
+          description: Internal error
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+
+components:
+  parameters:
+    ### PARAMETRI RICERCA PUBLIC KEY
+    pathKid:
```

Comment:
pathKid and pathVirtualKeyId are the same. merge them into pathKid

Reply:
Even though their type is the same (at least right now) they conceptually refer respectively to the public and virtual key id, that's why I preferred to keep them separated. Anyway, if you think that's a useless redundancy I can merge them in a single definition.